commands_cancelBuy.go

```diff
@@ -46,6 +46,20 @@ func (cb cancelBuyCmd) ToAuditEntry() string {
 	)
 }
 
+func cancelBuy() {
+
+}
```

Comment:
This should not have been included, it was accidentally committed.  Ill remove it.